org.captainjsparrow.steamdeckrepo.yml

```diff
@@ -0,0 +1,26 @@
+id: org.captainjsparrow.steamdeckrepo
```

Comment:
Is this an actual domain?

Reply:
https://steamdeckrepo.com/ is the root domain